packages/builder/src/pages/builder/portal/apps/_FirstAppOnboarding/PanelHeader.svelte

```diff
@@ -0,0 +1,55 @@
+<script>
+  import FontAwesomeIcon from "components/common/FontAwesomeIcon.svelte"
+
+  export let onBack = null
+  export let title = ""
+  export let subtitle = ""
+</script>
+
+<div>
+  <img
+    alt="Budibase Logo"
+    class="budibaseLogo"
+    src={"https://i.imgur.com/Xhdt1YP.png"}
+  />
+  <div class="headingAndBack">
+    {#if onBack}
+      <button on:click={onBack}>
+        <FontAwesomeIcon name="fa-solid fa-chevron-left" />
+      </button>
+    {/if}
+    <h1>{title}</h1>
+  </div>
+  <p>{subtitle}</p>
```

Comment:
We should use a BBUI Body component here to ensure consistent colour and font. You can use global CSS rules if you need to customise with margin bottom etc.